.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: false
+git:
+  depth: 3
+language: java
+# using `openjdk8` leads to javadoc errors
+jdk: oraclejdk8
+env:
+  global:
+  - TERM=dumb         # for gradle
```

Comment:
What do you think of `–console=plain` or `-Dorg.gradle.console=plain`?

https://discuss.gradle.org/t/make-console-plain-default/23514

Reply:
Using TERM leads to simpler command lines.